torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1069,6 +1069,39 @@ def sample_inputs_linalg_norm(op_info, device, dtype, requires_grad):
         return inputs
 
 
+def sample_inputs_batch_norm(op_info, device, dtype, requires_grad, **kwargs):
+    make_arg = partial(make_tensor, device=device, dtype=dtype, requires_grad=requires_grad)
+    make_arg_without_grad = partial(make_tensor, device=device, dtype=dtype, requires_grad=False)
```

Comment:
nit: "make arg without grad" sounds like "we're making an argument without a grad tensor", which confused me. We should rename it to something like "make_non_differentiable_arg" or "make_arg_without_requires_grad"

Reply:
Fixed. Sorry for the confusion :)